packages/@glimmer/reference/lib/iterable.ts

```diff
@@ -275,3 +278,40 @@ class ArrayIterator implements OpaqueIterator {
     return { key, value, memo };
   }
 }
+
+export class NativeIteratorDelegate<T = unknown> implements IteratorDelegate {
```

Comment:
This was necessary for testing the native iterator stuff. We should probably removed defaults in general from the environment